src/pages/hello.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
@yosriady Is this file supposed to exist? Looks like a hello world example to me :)

Reply:
Fixed.